src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -155,15 +156,27 @@
   text-transform: inherit; /* 2 */
   gap: 0.25em; /* 3 */
 
+  svg {
+    --icon-size-default: 1.25em; /* 4 */
+  }
+
   &:disabled,
   &:disabled:hover {
     color: var(--eds-theme-color-text-disabled);
   }
 }
 
-/* Style icons that may be in the clickable style children */
-.clickable-style--link svg {
-  --icon-size-default: 1.25em;
```

Comment:
I moved this into the main style block because that felt more consistent with the rest of the codebase.